research/conditional/moe_layers/continuous_moe.py

```diff
@@ -97,37 +95,31 @@ def get_temperature(self):
         return self.temperature, self.temperature
 
     def merge_map_emit(self, x, merge_weights, emit_weights):
-        # x shape is free_dimension, aggr_dimension, dmodel
-        # merge_weights shape is free_dimension, aggr_dimension // group_size, group_size, n_experts
+        # merge_weights shape is (free_dimension, aggr_dimension // group_size, group_size, n_experts)
+        # x shape is (free_dimension, aggr_dimension // group_size, group_size, dmodel)
```

Comment:
have x be the first here, also add `merge_weights and emit_weights shape is .,,,`

Reply:
Done